hedera-node/hedera-network-admin-service-impl/src/main/java/com/hedera/node/app/service/networkadmin/impl/handlers/NetworkTransactionGetRecordHandler.java

```diff
@@ -58,13 +70,73 @@ public Response createEmptyResponse(@NonNull final ResponseHeader header) {
     @Override
     public void validate(@NonNull final QueryContext context) throws PreCheckException {
         requireNonNull(context);
-        throw new UnsupportedOperationException("Not implemented");
+        final var op = context.query().transactionGetRecordOrThrow();
+
+        // The transaction ID must be specified
+        if (!op.hasTransactionID()) throw new PreCheckException(INVALID_TRANSACTION_ID);
+
+        // The record must exist for that transaction ID
+        final var txnId = op.transactionIDOrThrow();
```

Comment:
I guess this comment should be moved down because the follow line doesn't actually verify the record exists

Reply:
you are right I moved it base on Michael comment and forgot to grab the comment as well